contracts/MoMABurn.sol

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+import "./FeralfileArtworkV3_3.sol";
```

Comment:
I think we can just include the function we call so we do not need to include the whole contract.
This would save some cost for the deployment. For example,
```
contract FeralfileExhibitionV3_3 {
    function mintArtworkEdition(uint256 _artworkID, address _owner) public {}
}
```

Same for `ERC721`. We only calls few functions.

Reply:
Updated